weblate/templates/i18n.html

```diff
@@ -19,26 +19,26 @@
 {% blocktrans %}Use a few words, avoid common phrases.{% endblocktrans %}
 {% blocktrans %}No need for symbols, digits, or uppercase letters.{% endblocktrans %}
 {% blocktrans %}Add another word or two. Uncommon words are better.{% endblocktrans %}
-{% blocktrans %}Straight rows of keys are easy to guess.{% endblocktrans %}
-{% blocktrans %}Short keyboard patterns are easy to guess.{% endblocktrans %}
+{% blocktrans %}Straight rows of keys are easily guessed.{% endblocktrans %}
+{% blocktrans %}Short keyboard patterns are easily guessed.{% endblocktrans %}
 {% blocktrans %}Use a longer keyboard pattern with more turns.{% endblocktrans %}
-{% blocktrans %}Repeats like "aaa" are easy to guess.{% endblocktrans %}
-{% blocktrans %}Repeats like "abcabcabc" are only slightly harder to guess than "abc".{% endblocktrans %}
+{% blocktrans %}Repetitions like "aaa" are easily guessed.{% endblocktrans %}
+{% blocktrans %}Repetitions like "abcabcabc" are only slightly harder to guess than "abc".{% endblocktrans %}
 {% blocktrans %}Avoid repeated words and characters.{% endblocktrans %}
-{% blocktrans %}Sequences like "abc" or "6543" are easy to guess.{% endblocktrans %}
+{% blocktrans %}Sequences like "abc" or "6543" are easily guessed.{% endblocktrans %}
 {% blocktrans %}Avoid sequences.{% endblocktrans %}
-{% blocktrans %}Recent years are easy to guess.{% endblocktrans %}
+{% blocktrans %}Recent years are easily guessed.{% endblocktrans %}
 {% blocktrans %}Avoid recent years.{% endblocktrans %}
-{% blocktrans %}Avoid years that are associated with you.{% endblocktrans %}
+{% blocktrans %}Avoid years associated with you.{% endblocktrans %}
 {% blocktrans %}Dates are often easy to guess.{% endblocktrans %}
-{% blocktrans %}Avoid dates and years that are associated with you.{% endblocktrans %}
-{% blocktrans %}This is a top-10 common password.{% endblocktrans %}
-{% blocktrans %}This is a top-100 common password.{% endblocktrans %}
+{% blocktrans %}Avoid dates and years associated with you.{% endblocktrans %}
+{% blocktrans %}This is among the 10 most common passwords.{% endblocktrans %}
+{% blocktrans %}This is among the 100 most common passwords.{% endblocktrans %}
 {% blocktrans %}This is a very common password.{% endblocktrans %}
 {% blocktrans %}This is similar to a commonly used password.{% endblocktrans %}
-{% blocktrans %}A word by itself is easy to guess.{% endblocktrans %}
+{% blocktrans %}A word by itself is easily guessed.{% endblocktrans %}
 {% blocktrans %}Names and surnames by themselves are easy to guess.{% endblocktrans %}
-{% blocktrans %}Common names and surnames are easy to guess.{% endblocktrans %}
+{% blocktrans %}Common names and surnames are easily guessed.{% endblocktrans %}
```

Comment:
As noted in the comment, all above messages are coming from django-zxcvbn-password, the strings are extracted here for localization only. If you want to submit fixes to these, please do that at https://github.com/dwolfhub/zxcvbn-python/blob/master/zxcvbn/feedback.py